project/Dependencies.scala

```diff
@@ -44,4 +43,10 @@ object Dependencies {
     val effect          = "org.typelevel" %% "cats-effect"     % version
     val effectStd       = "org.typelevel" %% "cats-effect-std" % version
   }
+
+  object Smetrics {
+    private val version       = "1.0.5"
+    val smetrics              = "com.evolutiongaming" %% "smetrics"            % version
+    val `smetrics-prometheus` = "com.evolutiongaming" %% "smetrics-prometheus" % version
```

Comment:
why do we need impl?

Reply:
It's used by `KafkaMetricsCollector`, namely `io.prometheus.client.Collector` and friends. If we don't want to pull that into the main module, I suggest we move `KafkaMetricsCollector` into a separate module, does that sound good to you?

Reply:
yes, it does